### PR TITLE
[TTS] [bugfix] German FastPitch HiFi-GAN tutorial and lr

### PR DESCRIPTION
**Pull request by aroraakshit**
# What does this PR do ?

Address QA bugs for German TTS tutorial. Correct PR instead of https://github.com/NVIDIA/NeMo/pull/4449 

**Collection**: TTS

# Changelog 
- tutorial notebook changed
- learning rate updated to 1e-3 based on https://github.com/NVIDIA/NeMo/pull/4308 

**PR Type**: Bugfix

### CODE REVIEW

**Review by redoctopus — APPROVED** [review]

File: tutorials/tts/Fastpitch_Training_GermanTTS.ipynb [hunk 1]
```diff
@@ -566,51 +572,47 @@
...
+    "    train_dataset=../DataGermanTTS/thorsten-de/train_manifest_phonemes.json \\\n",
+    "    validation_datasets=../DataGermanTTS/thorsten-de/val_manifest_phonemes.json \\\n",
+    "    sup_data_path=../DataGermanTTS/thorsten-de/phonemes/ \\\n",
+    "    whitelist_path=./whitelist.tsv \\\n",
+    "    exp_manager.exp_dir=resultGermanTTS \\\n",
     "    trainer.max_epochs=1 \\\n",
-    "    pitch_mean=132.524658203125 \\\n",
-    "    pitch_std=37.389366149902344 \\\n",
+    "    pitch_mean=#paste_pitch_mean_here \\\n",
+    "    pitch_std=#paste_pitch_std_here \\\n",
```
Comment (rlangman) [inline tutorials/tts/Fastpitch_Training_GermanTTS.ipynb:607]: Would be cleaner to have the pitch mean/std be python variables to be overridden

**Review by rlangman — APPROVED** [review]
Looks good to me. Thanks!

File: tutorials/tts/Fastpitch_Training_GermanTTS.ipynb (hunk 1 above)
Comment (aroraakshit) [reply tutorials/tts/Fastpitch_Training_GermanTTS.ipynb:607]: Thanks for the feedback Ryan. Will improve in next iterations.
